index.html

```diff
@@ -62,164 +63,166 @@ <h5 class="subtitle">LET'S CONNECT</h5>
+            <div class="work-info">
+              <p class="work-info-title">CANOPY</p>
+              <span class="grey-dot"></span>
+              <p class="work-info-sub">Back End Dev</p>
+              <span class="grey-dot"></span>
+              <p class="work-info-sub">2015</p>
+            </div>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work-tags-div disable-default">
+              <li>
+                <p class="work-tags">html</p>
+              </li>
+              <li>
+                <p class="work-tags">css</p>
+              </li>
+              <li>
+                <p class="work-tags">javascript</p>
+              </li>
+            </ul>
+            <button outlined class="btn work-button">See Project</button>
           </div>
-          <p class="work-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+        </div>
+      </div>
+    </section>
+    <section class="container top-right-curved">
+      <div class="about left-right-margin">
+        <div class="about-myself">
+          <h1 class="about-headline">
+            About <br />
+            Myself
+          </h1>
+          <p class="about-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
           </p>
-          <ul class="work-tags-div disable-default">
+          <h5 class="subtitle">LET'S CONNECT</h5>
+          <ul class="social-links disable-default">
             <li>
-              <p class="work-tags">html</p>
+              <a>
+                <img alt="twitter" src="assets/twitter.svg" />
+              </a>
             </li>
             <li>
-              <p class="work-tags">css</p>
+              <a>
+                <img alt="linkedin" src="assets/linkedin.svg" />
+              </a>
             </li>
             <li>
-              <p class="work-tags">javascript</p>
+              <a>
+                <img alt="medium" src="assets/medium.svg" />
+              </a>
             </li>
-          </ul>
-          <button outlined class="work-button">See Project</button>
-        </div>
-        <div class="work-card">
-          <div class="title">
-            <img class="work-img" src="assets/sample-3.jpg" />
-            <h1 class="work-headline">Trees</h1>
-          </div>
-          <div class="work-info">
-            <p class="work-info-title">CANOPY</p>
-            <span class="grey-dot"></span>
-            <p class="work-info-sub">Back End Dev</p>
-            <span class="grey-dot"></span>
-            <p class="work-info-sub">2015</p>
-          </div>
```

Comment:
- [ ] According to the requirement left one takes 40% and the right one the remaining 60% of the page's width. Please adjust this in your About section.
- [ ] Also see the `arrow` on the right side and please fix it.
### Your design
![his](https://user-images.githubusercontent.com/81727988/220851985-5e96e6d2-73e9-4db7-98e3-c44c731c9c6b.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/220852117-49b395d0-17ea-45c1-98c6-44f6ff5754b4.png)